.github/workflows/reusable-publish-docs.yml

```diff
@@ -125,4 +125,16 @@ jobs:
         run: |
           aws s3 sync \
             site/ \
-            s3://${{ secrets.AWS_DOCS_BUCKET }}/lambda-typescript/${{ env.ALIAS }}/
\ No newline at end of file
+            s3://${{ secrets.AWS_DOCS_BUCKET }}/lambda-typescript/${{ env.ALIAS }}/
+      - name: Deploy Docs (Version JSON)
+        env:
+          VERSION: ${{ inputs.version }}
+          ALIAS: ${{ inputs.alias }}
+        run: |
+          aws s3 cp \
+            s3://${{ secrets.AWS_DOCS_BUCKET }}/lambda-typescript/versions.json \
+            versions_old.json
+          jq '. += [{"version": "${{ env.VERSION }}", "title": "${{ env.VERSION }}", "aliases": []}]' < versions_old.json > versions.json
```

Comment:
What about the `latest` alias?

Reply:
It's not needed, as far as I can see, the latest alias doesn't show in the list